docs/docs/user/core-concepts/defaults.md

```diff
@@ -1,6 +1,10 @@
 # Defaults
 
-KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](../../../resources/pipeline-defaults/defaults). This file provides the user with the power to set defaults for any and all [components](../components), thus omitting the need to repeat the same settings in [`pipeline.yaml`](../../../resources/pipeline-components/pipeline).
+KPOps has a very efficient way of dealing with repeating settings which
+manifests as [`defaults.yaml`](../../../resources/pipeline-defaults/defaults).
+This file provides the user with the power to set defaults for any and all [components](./components/overview.md),
+thus omitting the need to repeat the same settings in
+[`pipeline.yaml`](../../../resources/pipeline-components/pipeline).
```

Comment:
Why all the newlines?

Reply:
It was very hard to read the markdown file if everything is written in a single line.
This doesn't change the format of the docs it just makes it easier to read the raw file.

Reply:
did you enable line wrapping in your editor?

Reply:
I would prefer not to do manual linebreaks everywhere and instead rely on auto formatting

Reply:
Do we have a formatter that we use for markdown? Wouldn't it make sense to have a pre-commit check for that?
Something like: https://github.com/executablebooks/mdformat

Reply:
yeah, that would be a good idea. FYI in https://github.com/bakdata/ci-templates we're using dprint
https://github.com/bakdata/ci-templates/blob/main/.github/workflows/_format.yaml
https://github.com/bakdata/ci-templates/blob/main/dprint.json

Reply:
Ah! I was searching for it in the KPOPs repo! Thanks! 😄 I'll do a follow up PR for that!